test/SparqlExpressionTest.cpp

```diff
@@ -807,16 +807,26 @@ TEST(SparqlExpression, isSomethingFunctions) {
 
 // ____________________________________________________________________________
 TEST(SparqlExpression, testToNumericExpression) {
-  auto checkGetInt = testUnaryExpression<&makeIntExpression>;
-  auto checkGetDouble = testUnaryExpression<&makeDoubleExpression>;
-
-  checkGetInt(idOrLitOrStringVec({U, "  -1275", "5.97", "-78.97", "-5BoB6",
-                                  "FreBurg1", "", " .", " 42\n", " 0.01 ", ""}),
-              Ids{U, I(-1275), U, U, U, U, U, U, I(42), U, U});
-  checkGetDouble(
+  auto checkStrGetInt = testUnaryExpression<&makeIntExpression>;
+  auto checkStrGetDouble = testUnaryExpression<&makeDoubleExpression>;
+  auto checkNumGetInt = testUnaryExpression<&makeIntExpression>;
+  auto checkNumGetDouble = testUnaryExpression<&makeDoubleExpression>;
```

Comment:
If they are the completely same aliases, then you onlyneed two of them (checkToInt and checkToDouble), otherwise this is a little bit confusing.